components/suggest/src/store.rs

```diff
@@ -261,6 +259,18 @@ impl SuggestStore {
     ) -> SuggestApiResult<Option<SuggestProviderConfig>> {
         self.inner.fetch_provider_config(provider)
     }
+
+    #[handle_error(Error)]
+    pub fn fetch_geonames(
```

Comment:
Can we get a docstring for this API since this would be the only place for documentation?